components/card/src/vwc-card.scss

```diff
@@ -9,52 +9,79 @@
 	border-radius: 6px;
 	box-shadow: 0 4px 4px rgba(0, 0, 0, 0.05), 0 8px 8px rgba(0, 0, 0, 0.05), 0 2px 16px rgba(0, 0, 0, 0.1);
 	color: var(#{scheme-variables.$vvd-color-on-canvas});
-}
 
-.vwc-card-info {
-	display: flex;
-	flex-flow: column;
-	padding: 24px;
+	&-info {
+		display: flex;
+		flex-flow: column;
+		padding: 24px;
+	}
+
+	&-header {
+		display: flex;
+	}
+
+	&-title,
+	&-subtitle {
+		/* stylelint-disable property-no-vendor-prefix */
+		/* stylelint-disable value-no-vendor-prefix */
+		display: -webkit-box;
+		overflow: hidden;
+		-webkit-box-orient: vertical;
+		-webkit-line-clamp: var(--line-trim);
```

Comment:
I think there's a way to do this with sass. I mean a scoped sass variable rather than another confusing css variable.
alternatively we could just set the `-webkit-line-clamp` twice for each selector

Reply:
I'm not sure understood what you ment... :(